src/map/lua/lua_baseentity.cpp

```diff
@@ -17019,6 +17019,31 @@ bool CLuaBaseEntity::deleteRaisedChocobo()
     return true;
 }
 
+void CLuaBaseEntity::clearActionQueue()
+{
+    if (m_PBaseEntity == nullptr || m_PBaseEntity->objtype != TYPE_MOB)
+    {
+        return;
+    }
+
+    auto* PMob = static_cast<CMobEntity*>(m_PBaseEntity);
+
+    if (PMob)
+    {
+        PMob->PAI->ClearActionQueue();
+    }
+}
+
+void CLuaBaseEntity::clearTimerQueue()
```

Comment:
Have you tested this binding with a player being the target? Also, m_PBaseEntity will not be null for anything here, and that specific check is unnecessary

Reply:
You can cover all your bases here by checking for the presence of m_PBaseEntity->PAI too

Reply:
> Have you tested this binding with a player being the target? Also, m_PBaseEntity will not be null for anything here, and that specific check is unnecessary

Thanks for comments, I tried on player and it seemed ok, thus I removed the monster only check. I also removed the m_PBaseEntity null check and added just a m_PBaseEntity->PAI check.

Reply:
> You can cover all your bases here by checking for the presence of m_PBaseEntity->PAI too

Yep, I added that check now and removed the m_PBaseEntity check as per claywar.